tests/test_bgp_neighbor_af.rb

```diff
@@ -64,15 +132,15 @@ def clean_af(af_args, ebgp=true)
   # AF test matrix
   @@matrix = { # rubocop:disable Style/ClassVars
     # 1  => [1, 'default', '10:1::1', %w(ipv4 multicast)], # UNSUPPORTED
-    # 2  => [1, 'default', '10:1::1', %w(ipv4 unicast)],
+    # 2  => [1, 'default', '10:1::1', %w(ipv4 unicast)], # UNSUPPORTED in XR
     # 3  => [1, 'default', '10:1::1', %w(ipv6 multicast)],
     # 4  => [1, 'default', '10:1::1', %w(ipv6 unicast)],
     # 5  => [1, 'default', '1.1.1.1', %w(ipv4 multicast)],
     6  => [1, 'default', '1.1.1.1', %w(ipv4 unicast)],
     # 7  => [1, 'default', '1.1.1.1', %w(ipv6 multicast)],
-    8  => [1, 'default', '1.1.1.1', %w(ipv6 unicast)],
```

Comment:
Yeah I felt it wasn't adding much other than another whole cycle of running through the tests which are already pretty slow. So I went with: ipv4 uni default vrf, same with a real vrf and the l2vpn case. I'll put it back in if you want?